raphtory/src/graph.rs

```diff
@@ -146,55 +191,7 @@ impl Graph {
         }
     }
 
-    pub fn add_vertex(&self, t: i64, v: &PyAny, props: HashMap<String, Prop>) {
```

Comment:
why is this being removed to window graph?

Reply:
I just moved them around and added some comments - GitHub just being a little wonky